backend/routes/users.js

```diff
@@ -1,6 +1,6 @@
 const express = require("express");
 
-const Use = require('../models/user');
```

Comment:
R u sure?

Reply:
Yes. What's the problem here?

Reply:
Y use and not user

Reply:
because it's a variable name for the user export. 
Also I'm referencing it in a bunch of other places in the code and alex changed it and broke it.